examples/howto/model_builder.myst.md

```diff
@@ -271,8 +271,12 @@ After using the `predict()`, we can plot our data and see graphically how satisf
 fig, ax = plt.subplots(figsize=(7, 7))
 ax.plot(
     x_pred,
+    marker="x",
+    label="predict",
+),
+ax.plot(
```

Comment:
The changes are modifying an x vs y plot to two line plots where the x axis is always an arange and the y axis is mapped to the values in the 1st plot and to the y values in the 2nd plot. Mapping these two different quantities to the same axis is definitely wrong and should be changed. I think what @fonnesbeck wanted in the issue was having both scatter x vs y of the predictions along with the x vs y of the regressions lines from the model that generated those predictions. It would have a similar look to the last image in https://www.pymc.io/projects/examples/en/latest/generalized_linear_models/GLM-robust.html but with the x_pred / pred_means["y"] as the scatter instead of truth valued and no true regression line

Reply:
Ok, I get the picture now, I will work on it